lib/event.js

```diff
@@ -11,6 +11,14 @@ module.exports = {
     passed: 'test.passed',
     failed: 'test.failed',
   },
+  hook: {
```

Comment:
we can't use test.after/suite.after, because they are already defined. I suggest test.afterError, suite.afterError